src/python_framework_v02/troute/nhd_io.py

```diff
@@ -380,7 +380,7 @@ def process_one_path(path):
             return ds
 
     datasets = [process_one_path(p) for p in paths]
-    combined = xr.concat(datasets, dim)
+    combined = xr.concat(datasets, dim, combine_attrs = "override")
```

Comment:
this extra argument is the default of xr.concat. However, it ensures that only the attribute of the FIRST entry in the datasets list is passed as the attribute. In this case, because paths are processed in temporal order, it is the sliceCenterTimeUTC of the very first TimeSlice file. So, if we glob files between 2018-09-18_00:00 and 2018-09-18_09:45, then the sliceCenterTimeUTC attribute listed in combined is 2018-09-18_00:00. This might not work properly if globbed files are processed out of temporal order.